src/ocm/components/AddHosts/utils.ts

```diff
@@ -1,15 +1,32 @@
 import { isSNO } from '../../../common';
-import { canAddHostSNO } from '../../../common/utils';
 import Day2ClusterService from '../../services/Day2ClusterService';
+import {
+  getFeatureSupportLevelsMap,
+  getVersionSupportLevelsMap,
+} from '../featureSupportLevels/FeatureSupportLevelProvider';
 import { OcmClusterType } from './types';
 
 export const canAddHost = ({ cluster }: { cluster: OcmClusterType }) => {
+  const featureSupportLevelsMap =
```

Comment:
We could extract all this logic to a separate function. Following our new approach, It could be a `clusterSelector` given it needs the cluster and the feature to check.

`
const isSNOExpansionSupported = hasSupportedFeature(cluster, 'SINGLE_NODE_EXPANSION');
`

Reply:
The `cluster` here isn't really the cluster we use in clusterSelectors. I'd rather keep it separate.

Reply:
True, that makes sense